apps/ewallet/lib/ewallet/web/v1/auth/client_auth.ex

```diff
@@ -0,0 +1,41 @@
+defmodule EWallet.Web.V1.ClientAuth do
```

Comment:
**Future refactor:** Thinking that more can be done to tidy up the auth-related code